scalafix-core/src/main/scala/scalafix/rewrite/NoValInForComprehension.scala

```diff
@@ -0,0 +1,21 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import scala.meta.contrib._
+
+case object NoValInForComprehension extends Rewrite {
+
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    ctx.tree.collect {
+      case f: Term.ForYield =>
```

Comment:
Right, I actually though of it, but I couldn't remember whether a `for` without `yield` were valid scala syntax.
Matching `Enumerator.Val` could actually do the trick!